website/docs/layouts/flex/partials/code/how-to-use.md

```diff
@@ -0,0 +1,573 @@
+## How to use this component
+
+!!! Insight
+
+It's much easier to use this component if one is familiar with the CSS3 Flexible Box specifications.
+<br/>
+A good and quick introduction to flexbox can be found in [this MDN guide](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox).
+
+!!!
+
+The `Layout::Flex` and optional `Layout::Flex::Item` components provide a way to quickly build out flexbox-based layouts of components or elements without needing to write a lot of custom CSS code or understand all the intricacies of CSS flexbox styles.
+
+### Basic usage
+
+The simplest way to implement a flexbox layout is by using the `Layout::Flex` component to wrap some content:
+
+```handlebars{data-execute=false}
+<Hds::Layout::Flex>
+  <div>{{! some content here }}</div>
+  <div>{{! some other content here }}</div>
+  <div>{{! more content here }}</div>
+</Hds::Layout::Flex>
+```
+
+In this way, every direct child element of the **flexbox container** will be treated, by the browser's layout engine, as a **flex item** (for details about what this means, refer to the guide linked at the top of this page).
+
+There are cases in which is necessary to wrap one or more child elements in a specific `Layout::Flex::Item` (eg. to apply the `@basis/@grow/@shrink` arguments, see below for details):
+
+```handlebars{data-execute=false}
+<Hds::Layout::Flex as |LF|>
+  <div>{{! some content here }}</div>
+  <LF.Item @grow={{false}}>
+    {{! some other content here }}
+  </LF.Item>
+  <div>{{! more content here }}</div>
+</Hds::Layout::Flex>
+```
+
+!!! Info
+
+Note: there is no strict need to use the `Layout::Flex::Item` subcomponent as a direct child of `Layout::Flex`; use it only when necessary to tweak flex styles of individual child item via the `@basis/@grow/@shrink` (to avoid rendering an extra Ember component).
```

Comment:
```suggestion
Note: there is no strict need to use the `Layout::Flex::Item` subcomponent as a direct child of `Layout::Flex`; use it only when necessary to tweak flex styles of an individual child item via the `@basis/@grow/@shrink` (to avoid rendering an extra Ember component).
```